compiler/src/iree/compiler/Codegen/SPIRV/Passes.cpp

```diff
@@ -44,6 +44,11 @@
 namespace mlir {
 namespace iree_compiler {
 
+static llvm::cl::opt<int> clSPIRVIndexingBits(
+    "iree-spirv-index-bits",
```

Comment:
Can we use `iree-spirv-index-bitwidth` to be clear?

Reply:
I figured it would be better to be consistent with the other index related flags. Maybe they should all be updated then?
```
--iree-stream-resource-index-bits=<uint>
--iree-vm-target-index-bits=<int>
```

Reply:
Ah, okay, forgot about those. Makes sense.